api/base/filters.py

```diff
@@ -148,6 +148,19 @@ def _parse_date_param(self, field, field_name, op, value):
                 'value': stop
             }]
 
+    def bulk_get_values(self, value, field):
+        """
+        Returns list of values from query_param for IN query
+
+        If url contained `/nodes/?filter[id]=12345, abcde`, the returned values would be:
+        [u'12345', u'abcde']
+        """
+        if value[0] == '[' and value[-1] == ']':
+            value = value.replace('[', '').replace(']', '')
```

Comment:
This is better.  I don't want the user to put brackets around the ids and the documentation says to put `filter[id]=12345,abcde`, but I think it's likely someone tries it with brackets.   I wanted to handle this case, but I also don't want brackets to be stripped from a title with brackets, for example.